README.md

```diff
@@ -213,6 +232,26 @@ module.exports = {
 };
 ```
 
+### `bumperTagOpts`
+
+This option will be passed as the first argument to
+[`bumper.tag`](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-recommended-bump/README.md#api)
+
+### `bumperCommitsOpts`
+
+This option will be passed as the first argument to
+[`bumper.commits`](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-recommended-bump/README.md#api)
+
+### `bumperParserOpts`
+
+This option will be passed as the second argument to
+[`bumper.parserOptions`](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-recommended-bump/README.md#api)
+
+### `whatBump`
```

Comment:
Nitpick:
If you're going with `bumper` prefix for other options (to scope them to underlying `bumper` package in a way, right?) then I think this one should be called:
```suggestion
### `bumperWhatBump`
```
just to follow the convention 🤷

Reply:
Agreed, actually prefer unprefixed but then we'd have `parserOpts` 🙄 but they might actually be the same thing in conventional-changelog land, I'll double-check that.